maze_transformer/training/config.py

```diff
@@ -149,6 +156,20 @@ class TrainConfig(SerializableDataclass):
         print_loss_interval=50,
         checkpoint_interval=10000,
     ),
+    TrainConfig(
```

Comment:
Is this supposed to be in here? Seems unrelated.

Reply:
yeah, you're right. that was when I was also planning to include another trained model with multiple checkpoints in the repo, but then decided this should wait until after #157 and forgot about it. I can remove it, but it shouldn't cause any issues for the moment.